src/main/java/com/jcabi/github/RtUser.java

```diff
@@ -139,6 +147,29 @@ public UserEmails emails() {
     }
 
     @Override
+    public List<Notification> notifications() throws IOException {
+        final List<Notification> notifs =
+            new LinkedList<Notification>();
+        final JsonResponse resp = this.github().entry().uri()
+            .path("notifications")
+            .back()
+            .fetch()
+            .as(JsonResponse.class);
+        final JsonArray jsnnotifs = resp.json().readArray();
+        for (final JsonValue jsnnotif : jsnnotifs) {
```

Comment:
there are three variables in this method and they all have very strange names. why so? How about `list`, `array` and `value`? Read this http://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html